bundles/org.openhab.binding.ipcamera/src/main/java/org/openhab/binding/ipcamera/internal/handler/IpCameraHandler.java

```diff
@@ -1664,6 +1684,9 @@ public void initialize() {
         snapshotUri = getCorrectUrlFormat(cameraConfig.getSnapshotUrl());
         mjpegUri = getCorrectUrlFormat(cameraConfig.getMjpegUrl());
         rtspUri = cameraConfig.getFfmpegInput();
+        if (cameraConfig.getFfmpegOutput().isEmpty()) {
+            cameraConfig.setFfmpegOutput("/etc/openhab/html/cameras/" + this.thing.getUID().getId() + "/");
```

Comment:
It doesn't feel right to hard code this path, although it's a default value and it doesn't feel right to put video data in the /etc folder, either. Can you elaborate on this?

Reply:
Happy to change the location, it just needs to have write permissions. Do you have a suggestion on where it can go?

Reply:
The userdata folder is intended for such a use case. See https://github.com/openhab/openhab-addons/blob/e24d43aecbcfc5af551c8e9087d9be6992e5c69a/bundles/org.openhab.binding.playstation/src/main/java/org/openhab/binding/playstation/internal/PS4ArtworkHandler.java#L57 for example.

Reply:
DONE: thanks for suggesting as that is a better way. Readme is updated to reflect changes.